drop-in/src/test/java/com/adyen/checkout/dropin/DropInConfigurationTest.kt

```diff
@@ -0,0 +1,143 @@
+package com.adyen.checkout.dropin
+
+import com.adyen.checkout.adyen3ds2.Adyen3DS2Configuration
+import com.adyen.checkout.adyen3ds2.getAdyen3DS2Configuration
+import com.adyen.checkout.card.CardConfiguration
+import com.adyen.checkout.card.getCardConfiguration
+import com.adyen.checkout.components.core.Amount
+import com.adyen.checkout.components.core.AnalyticsConfiguration
+import com.adyen.checkout.components.core.AnalyticsLevel
+import com.adyen.checkout.components.core.CheckoutConfiguration
+import com.adyen.checkout.core.Environment
+import com.adyen.checkout.googlepay.GooglePayConfiguration
+import com.adyen.checkout.googlepay.getGooglePayConfiguration
+import com.adyen.checkout.ideal.IdealConfiguration
+import com.adyen.checkout.ideal.getIdealConfiguration
+import com.adyen.checkout.redirect.RedirectConfiguration
+import com.adyen.checkout.redirect.getRedirectConfiguration
+import com.adyen.checkout.wechatpay.WeChatPayActionConfiguration
+import com.adyen.checkout.wechatpay.getWeChatPayActionConfiguration
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertNotNull
+import org.junit.jupiter.api.Test
+import java.util.Locale
+
+internal class DropInConfigurationTest {
```

Comment:
I have a suggestion for 1 more scenario to test:

```kotlin
val checkoutConfiguration = CheckoutConfiguration() {
    dropIn {
        ...
    }
    card {
        ...
    }
    someOtherPM {
        ...
    }
}
// assert drop-in and component configs are added
```

Maybe it can be combined with the first test?
I tried to think of more use cases to test but I'm not sure how practical they are so I'll leave it up to you to decide if there are other valid use cases to test.

Reply:
Hmm I don't think that's in scope for this test. Payment method configs are not added to the drop-in config when using `CheckoutConfiguration`. So the tests for the respective payment method configs should cover these cases

Reply:
Ah good one 👍